bindings/java/devdoc/java_binding_hld.md

```diff
@@ -145,9 +154,10 @@ gateway, it:
 -   Constructs a `Broker` Java object using the `BROKER_HANDLE`.
 
 -   Finds the module’s class with the name specified by the `args.class_name`,
-    invokes the constructor passing in the native `MODULE_HANDLE` address,
-    the `Broker` object and the JSON args string for that module, and 
-    creates the Java module.
+    finds the constructor that matches these three arguments: the native `MODULE_HANDLE` address,
+    the `Broker` object and the JSON args string for that module and invokes it. 
+    If the constructor with three parameters is not found, finds no-argument constructor and invokes it and 
+    invokes `create` method implemented by the Java module.
```

Comment:
1. I think the user wouldn't call create from no-argument constructor because it doesn't have the objects to pass for address, broker, configuration.
2. OK
3. If they have the constructor with 3 params it will call it, otherwise no-argument constructor is required.